src/build/app/mod.rs

```diff
@@ -1508,7 +1511,7 @@ impl<'b> App<'b> {
     // used in clap_generate (https://github.com/clap-rs/clap_generate)
     #[doc(hidden)]
     pub fn _build(&mut self) {
-        debugln!("App::_build;");
```

Comment:
I won't be commenting on each occasion separately, but: please, do not remove the struct names. I love them, I really do.

Reply:
In modules that have more than one struct, definitely. But we don't need to write this info when that specific module has only 1 struct. Here, `clap::build::app` is the module name and it has only 1 struct `App`.

Reply:
The thing is, I can *grep* for `Arg::_debug_asserts` or stuff. Copy, click "search in folder", paste, click, end up in the exact place I need. With the new system, I need to manually _navigate_ to the module location and only then I can grep (and probably end up with a few more matches than it used to be in the old system).

Reply:
And to address the "only one struct" point - currently it's true, but we've been talking about factoring things out into `*Base` or stuff. We can very well end up with a few structs in future.